fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1593,10 +1593,10 @@ Future<RangeResult> TracingOptionsImpl::getRange(ReadYourWritesTransaction* ryw,
 
 		if (key.endsWith(kTracingTransactionIdKey)) {
 			result.push_back_deep(result.arena(),
-			                      KeyValueRef(key, std::to_string(ryw->getTransactionState()->spanID.first())));
+			                      KeyValueRef(key, ryw->getTransactionState()->spanContext.traceID.toString()));
```

Comment:
I assume the deserialization in the unit tests is passing now? What did you have to do to get it working other than commenting out some of the tests?

Reply:
There was another test failing in the client unit tests and I missed it. I ended up commenting them out and then adding them back fixing, one by one until they were all enabled.